modules/widgets/customizer-utils.js

```diff
@@ -0,0 +1,77 @@
+/* global gapi, FB, twttr */
+
+/**
+ * Utilities to work with widgets in Customizer.
+ */
+
+/**
+ * Checks whether this Customizer supports partial widget refresh.
+ * @returns {boolean}
+ */
+wp.customizerHasPartialWidgetRefresh = function() {
+	return 'object' === typeof wp && 'function' === typeof wp.customize && 'object' === typeof wp.customize.selectiveRefresh && 'object' === typeof wp.customize.widgetsPreview && 'function' === typeof wp.customize.widgetsPreview.WidgetPartial;
+};
+
+/**
+ * Verifies that the placed widget ID contains the widget name.
+ * @param {object} placement
+ * @param {string} widgetName
+ * @returns {*|boolean}
+ */
+wp.isJetpackWidgetPlaced = function( placement, widgetName ) {
+	var regex = new RegExp( '^' + widgetName + '-\\d+$', 'g' );
```

Comment:
Is it necessary to check for a digit after the widget name? Can we just use a string match and make sure that `widgetName` is strictly in the beginning?